l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -227,9 +232,9 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
             // slither-disable-next-line arbitrary-send-eth
             sharedBridge.bridgehubDepositBaseToken{value: msg.value}(
                 _request.chainId,
+                IL1NativeTokenVault(sharedBridge.nativeTokenVault()).getAssetInfoFromLegacy(token),
                 msg.sender,
-                token,
-                _request.mintValue
+                _request.mintValue - msg.value
```

Comment:
better to add a comment on why when we are using ETH this number should be 0 and full `mintValue` otherwise

Reply:
moved the subtraction to the bridgehubDepositBaseToken